app/models/package_manager/go.rb

```diff
@@ -52,8 +53,20 @@ def self.project_names(since = 1.day.ago)
         .map { |line| JSON.parse(line)["Path"] }
     end
 
-    def self.update(name)
-      super(name)
+    def self.one_version(name, version_string)
+      info = get("#{PROXY_BASE_URL}/#{name}/@v/#{version_string}.info")
+      # Store nil published_at for known Go Modules issue where case-insensitive name collisions break go get
+      # e.g. https://proxy.golang.org/github.com/ysweid/aws-sdk-go/@v/v1.12.68.info
+      version_string = info.nil? ? version_string : info["Version"]
+      published_at = info && info["Time"].presence && Time.parse(info["Time"])
+      {
+        number: version_string,
+        published_at: published_at,
```

Comment:
Hey @mellisio, thoughts on trying to avoid this if we send published_at to the job from depper?

Reply:
Unless we intend to start sending the license too I don't think it matters. One thing we could do would be to send published_at, number, license then we wouldn't even have to hit an API on libraries side.